colors/PaperColor.vim

```diff
@@ -1232,840 +1232,842 @@ fun! s:apply_syntax_highlightings()
-  exec 'hi cssUnitDecorators' . s:fg_aqua
-  exec 'hi cssColor' . s:fg_blue . s:ft_bold
-  exec 'hi cssBraces' . s:fg_pink
-  exec 'hi cssBackgroundProp' . s:fg_foreground
-  exec 'hi cssTextProp' . s:fg_foreground
-  exec 'hi cssDimensionProp' . s:fg_foreground
-  exec 'hi cssClassName' . s:fg_pink
-
-  " Markdown Highlighting
-  exec 'hi markdownHeadingRule' . s:fg_pink . s:ft_bold
-  exec 'hi markdownH1' . s:fg_pink . s:ft_bold
-  exec 'hi markdownH2' . s:fg_orange . s:ft_bold
-  exec 'hi markdownBlockquote' . s:fg_pink
-  exec 'hi markdownCodeBlock' . s:fg_olive
-  exec 'hi markdownCode' . s:fg_olive
-  exec 'hi markdownLink' . s:fg_blue . s:ft_bold
-  exec 'hi markdownUrl' . s:fg_blue
-  exec 'hi markdownLinkText' . s:fg_pink
-  exec 'hi markdownLinkTextDelimiter' . s:fg_purple
-  exec 'hi markdownLinkDelimiter' . s:fg_purple
-  exec 'hi markdownCodeDelimiter' . s:fg_blue
-
-  exec 'hi mkdCode' . s:fg_olive
-  exec 'hi mkdLink' . s:fg_blue . s:ft_bold
-  exec 'hi mkdURL' . s:fg_comment
-  exec 'hi mkdString' . s:fg_foreground
-  exec 'hi mkdBlockQuote' . s:fg_foreground . s:bg_popupmenu_bg
-  exec 'hi mkdLinkTitle' . s:fg_pink
-  exec 'hi mkdDelimiter' . s:fg_aqua
-  exec 'hi mkdRule' . s:fg_pink
-
-  " reStructuredText Highlighting
-  exec 'hi rstSections' . s:fg_pink . s:ft_bold
-  exec 'hi rstDelimiter' . s:fg_pink . s:ft_bold
-  exec 'hi rstExplicitMarkup' . s:fg_pink . s:ft_bold
-  exec 'hi rstDirective' . s:fg_blue
-  exec 'hi rstHyperlinkTarget' . s:fg_green
-  exec 'hi rstExDirective' . s:fg_foreground
-  exec 'hi rstInlineLiteral' . s:fg_olive
-  exec 'hi rstInterpretedTextOrHyperlinkReference' . s:fg_blue
-
-  " Python Highlighting
-  exec 'hi pythonImport' . s:fg_pink . s:ft_bold
-  exec 'hi pythonExceptions' . s:fg_red
-  exec 'hi pythonException' . s:fg_purple . s:ft_bold
-  exec 'hi pythonInclude' . s:fg_red
-  exec 'hi pythonStatement' . s:fg_pink
-  exec 'hi pythonConditional' . s:fg_purple . s:ft_bold
-  exec 'hi pythonRepeat' . s:fg_purple . s:ft_bold
-  exec 'hi pythonFunction' . s:fg_aqua . s:ft_bold
-  exec 'hi pythonPreCondit' . s:fg_purple
-  exec 'hi pythonExClass' . s:fg_orange
-  exec 'hi pythonOperator' . s:fg_purple . s:ft_bold
-  exec 'hi pythonBuiltin' . s:fg_foreground
-  exec 'hi pythonDecorator' . s:fg_orange
-
-  exec 'hi pythonString' . s:fg_olive
-  exec 'hi pythonEscape' . s:fg_olive . s:ft_bold
-  exec 'hi pythonStrFormatting' . s:fg_olive . s:ft_bold
-
-  exec 'hi pythonBoolean' . s:fg_green . s:ft_bold
-  exec 'hi pythonExClass' . s:fg_red
-  exec 'hi pythonBytesEscape' . s:fg_olive . s:ft_bold
-  exec 'hi pythonDottedName' . s:fg_purple
-  exec 'hi pythonStrFormat' . s:fg_foreground
-
-  if s:langOpt_python__highlight_builtins == 1
-    exec 'hi pythonBuiltinFunc' . s:fg_blue
-    exec 'hi pythonBuiltinObj' . s:fg_red
-  else
-    exec 'hi pythonBuiltinFunc' . s:fg_foreground
-    exec 'hi pythonBuiltinObj' . s:fg_foreground
-  endif
+  if !exists('g:only_generic_hl') || !g:only_generic_hl
```

Comment:
This variable has to be specifically set before using the colorscheme or it will do nothing. This behavior secures that nothing will change for users who don't specifically want it to.